docs/components/buttongroup/index.md

```diff
@@ -7,25 +7,26 @@
 Button Group is used to group and align clickable elements, applying different visuals for different needs.
 
 <components-status vue='released' />
+
 ## Example
 
 <theme-switcher />
 
 <buttongroup-example />
 
 ## Usage
-This component is only supported in Warp Vue at the moment.
 
+This component is only supported in Warp Vue at the moment.
 For a React equivalent that serves a similar purpose, use the `Toggle` in [RadioButtons](/components/radiobuttons/).
 
+<component-design-guidelines name="Warp - Components / Button Group" link="https://www.figma.com/file/nkiRpuVu6XRfvY96BA80H8/Components-overview?type=design&node-id=384-34744&mode=design" />
```

Comment:
Just so we are aware, this Figma link shows what Toggle of type radiobutton looks like ([example](https://warp-ds.github.io/tech-docs/components/radiobuttons/)). The ButtonGroup component only shows gray colors and has smaller border radius.

Reply:
This also belongs to the discussion here: https://sch-chat.slack.com/archives/C050N3NQ204/p1695892295336879
So I'm not sure about what to do with this right now? 🤷 😅

Reply:
I removed the link to Figma for now